lib/discovery.js

```diff
@@ -240,6 +242,19 @@ function mixinDiscovery(MySQL, mysql) {
     }
     table = args.table;
     options = args.options;
+
+    var default_options = {
```

Comment:
Feel free to leave it, the rule is wrong and shouldn't be there. I will submit another PR to fix later (I don't want you to have to rebase again if I submit the fix now).